projects/ion/src/stories/switch.stories.ts

```diff
@@ -0,0 +1,100 @@
+import { IonSwitchComponent } from '../lib/switch';
+
+import type { Meta, StoryObj } from '@storybook/angular';
+
+const meta: Meta<IonSwitchComponent> = {
+  title: 'Ion/Data Entry/Switch',
+  component: IonSwitchComponent,
+  tags: ['autodocs'],
+  render: (args: IonSwitchComponent) => ({
+    props: {
+      ...args,
+    },
+  }),
+  argTypes: {
+    key: {
+      control: {
+        type: 'text',
+      },
+      description: 'Chave do switch.',
+    },
+    value: {
+      control: {
+        type: 'boolean',
+      },
+      defaultValue: { summary: false },
+      description: 'Valor do switch.',
+    },
+    size: {
+      control: {
+        type: 'select',
+      },
+      options: ['sm', 'md', 'lg'],
+      defaultValue: { summary: 'sm' },
+      description: 'Tamanho do switch. Pode ser `sm`, `md` ou `lg`.',
+    },
+    disabled: {
+      control: {
+        type: 'boolean',
+      },
+      defaultValue: { summary: false },
+      description: 'Desabilita o switch.',
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<IonSwitchComponent>;
+export const Default: Story = {
+  args: {
+    key: 'Default',
+    value: false,
+    size: 'sm',
```

Comment:
Values who don't have difference to the default values, don't need be declared here.